src/services/photos/index.ts

```diff
@@ -60,17 +62,13 @@ export class PhotosService {
     limit: number;
     skip?: number;
   }): Promise<{ results: photos.Photo[]; count: number }> {
-    this.checkModel();
-
-    const { results, count } = await PhotosCommonServices.sdk.photos.getPhotos({}, skip, limit);
+    const { results, count } = await this.sdk.photos.photos.getPhotos({}, skip, limit);
 
     return { results, count };
   }
 
   public async deletePhotos(photos: photos.Photo[]): Promise<void> {
-    this.checkModel();
-
-    Promise.all(photos.map(async (photo) => PhotosCommonServices.sdk.photos.deletePhotoById(photo.id)));
+    Promise.all(photos.map(async (photo) => await this.sdk.photos.photos.deletePhotoById(photo.id)));
```

Comment:
`await` still missing

Reply:
https://gyazo.com/ab14619ab1e46476edee257eaffdc434 ?

Reply:
`await Promise.all(photos.map(async (photo) => await this.sdk.photos.photos.deletePhotoById(photo.id)));`